package.json

```diff
@@ -11,8 +11,8 @@
     "lint:node": "yarn eslint ./src --cache --ext ts --config .eslintrc",
     "format-lint": "yarn format && yarn lint:node",
     "build-clean": "rm -rf build",
-    "build-incremental": "yarn tsc --incremental && bash ./scripts/symlinkPlugin.sh && OCLIF_COMPILATION=true yarn oclif-dev manifest",
-    "build": "yarn build-clean && yarn tsc && OCLIF_COMPILATION=true yarn oclif-dev manifest",
+    "build-incremental": "yarn tsc --incremental && bash ./scripts/symlinkPlugin.sh && yarn oclif-dev manifest",
```

Comment:
The ideia of `OCLIF_COMPILATION` is to handle the functions call in the root of some modules, right ?

Reply:
Exactly